README.md

```diff
@@ -212,12 +214,52 @@ sudo rpm-ostree rebase ostree-image-signed:docker://ghcr.io/ublue-os/IMAGE:TAG
 
 It's a good idea to become familar with the [Fedora CoreOS Documentation](https://docs.fedoraproject.org/en-US/fedora-coreos/) as well as the [CoreOS rpm-ostree docs](https://coreos.github.io/rpm-ostree/). Note especially, this image is only possible due to [ostree native containers](https://coreos.github.io/rpm-ostree/container/).
 
-### Immutability and Podman
+### Podman
+
+#### Immutability and Podman
 
 A CoreOS root filesystem system is immutable at runtime, and it is not recommended to install packages like in a mutable "normal" distribution.
 
 Fedora CoreOS expects the user to run services using [podman](https://podman.io). `moby-engine`, the free Docker implementation, is also installed for those who desire docker instead of podman.
 
+#### Docker/Moby and Podman
+
+> [!IMPORTANT]
+> CoreOS [cautions against](https://docs.fedoraproject.org/en-US/fedora-coreos/faq/#_can_i_run_containers_via_docker_and_podman_at_the_same_time) running podman and docker containers at the same time.  Thus, `docker.socket` is disabled by default to prevent accidental activation of the docker daemon, given podman is the default.
+>
+> Ony run both simultaneously if you understand the risk.
+
+#### Podman and FirewallD
+
+Podman and firewalld [can sometimes conflict](https://github.com/ublue-os/ucore/issues/90) such that a `firewall-cmd --reload` removes firewall rules generated by podman.
+
+As of [netavark v1.9.0](https://blog.podman.io/2023/11/new-netavark-firewalld-reload-service/) a service is provided to handle re-adding netavark (Podman) firewall rules after a firewalld reload occurs.  If needed, enable like so: `systemctl enable netavark-firewalld-reload.service`
+
+#### Automatically start containers on boot
+
+By default, UCore does not automatically start `restart: always` containers on system boot, however this can be easily enabled:
+
+```bash
+# Copy the system's podman-restart service to the user location
+cp /lib/systemd/system/podman-restart.service /var/home/core/.config/systemd/user
+
+# Enable the user service
+systemctl --user enable podman-restart.service
```

Comment:
either before or after describing this for user run podman, could you also add the global?

Thanks!

Reply:
Sorry, what do you mean? What is "the global"?

The system (non-user) `podman-restart` service doesn't do anything useful whether it is enabled or disabled.

Reply:
Yes, by global, i mean system... `podman-restart` does the same thing for system as it does for users... but for starting `restart: always` rootful podman containers.

Or am I missing something?

Reply:
Thanks I will do. I'm not being argumentative - I literally only started using Podman and Docker when I started using Aurora, and it's all brand new to me.

Reply:
> Thanks I will do. I'm not being argumentative

No worries! I think your suggestion is very helpful, as clearly it's not always easy to find something that seems like it should be obvious (especially when more momentum of documentation seems to be around docker and docker-compose). 

I just think your suggestion applies equally well to both root/user podman operation. :-)

> I literally only started using Podman and Docker when I started using Aurora, and it's all brand new to me. I have zero clue what is the right way to do things.

You have much more than zero clue!  You are correct in your belief that many advocate for "only run podman as user", but that's not because podman itself is less secure than anything else, it's just to encourage more secure practices. So if you can and are willing to run all rootless, excellent! 

And again, I thank you for your contribution.